### ISSUE TITLE
:open command not working with NoScript active

### ISSUE THREAD

**Issue by ptrcarta** [vimperator/vimperator-labs#802]
##### Issue type:

- Bug report

##### Version:

```
Vimperator:	3.16.0 (created: 2017/02/01 00:55:15)
Firefox:	Mozilla/5.0 (X11; Linux x86_64; rv:52.0) Gecko/20100101 Firefox/52.0
NoScript: 5.1.2
```

##### Description:

open command doesn't work. tabopen instead works fine. As an example when typing `:open about:mozilla` and pressing enter,  the command bar disappears without errors, but the current page stays the same. This is not the case for winopen or tabopen.
Disabling NoScript solves the problem

**Comment by ptrcarta** [comment]
update: this happened after passing from firefox 56 to firefox-esr which is 52. Apparently with a clean profile the issue doesn't persist

**Comment by ptrcarta** [comment]
update: the issue is only present when NoScript is also active, but I'm not able to pinpoint the cause

**Comment by synkka** [comment]
you're the best! i thought i was taking crazy pills!

**Comment by krigstask** [comment]
> this happened after passing from firefox 56 to firefox-esr which is 52

Happens also on Firefox 52.0 ESR which was never updated to later versions.

Seems like a NoScript problem we'd better report.

**Comment by krigstask** [comment]
[NoScript 5.1.3rc1](https://noscript.net/?ver=5.1.3rc1&prev=5.1.2) fixed the issue for me.

**Comment by randomizedthinking** [comment]
@krigstask thanks... it solves my problem too. Just downgrade to ESR to continue using vimperator, :).

**Comment by SammysHP** [comment]
This seems to be a problem with NoScript, I had the same problem a few days ago and fixed it by updating Firefox to 56.0.1 and also update NoScript.
